validation/src/test/scala/hmda/validation/rules/lar/macro/Q065Spec.scala

```diff
@@ -0,0 +1,47 @@
+package hmda.validation.rules.lar.`macro`
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+import org.scalacheck.Gen
+
+class Q065Spec extends MacroSpec {
+
+  val threshold = config.getInt("hmda.validation.macro.Q065.hoepaLoanLimit")
+
+  def relevantLar(lar: LoanApplicationRegister): LoanApplicationRegister = {
+    lar.copy(hoepaStatus = 1)
+  }
+  def irrelevantLar(lar: LoanApplicationRegister): LoanApplicationRegister = {
+    lar.copy(hoepaStatus = 2)
+  }
+
+  def nonHoepaLoans: Gen[Int] = Gen.choose(0, 1000)
+
+  property(s"be valid if there are fewer than $threshold HOEPA loans") {
+    forAll(nonHoepaLoans) { extras =>
+      val belowThreshold = threshold - 1
+      val lars = larNGen(belowThreshold + extras).sample.get
```

Comment:
That strategy would mean that we can't test with 0 non-HOEPA loans for two of the specs. I like that my strategy proves that the edit still works when there's 0. 

Your strategy would make it marginally cleaner. Is that worth changing the testing strategy? I'm not sure.